controllers/selectMainRoute.js

```diff
@@ -23,26 +24,45 @@
 
       // list route is got from server;
       $scope.routes = []
-
       // the id of main route
       $scope.mainRoute = 0
-
+        $scope.setAsHomePage = function (model,$event) {
+            var confirm = $mdDialog.confirm()
+                .title('Set As Home Page')
+                .textContent('Are you sure you want to set this current page as your main home page that people land on when they visit your domain ?')
+                .targetEvent($event)
+                .ok('Confirm')
+                .cancel('Cancel');
+            $mdDialog.show(confirm).then(function() {
+                model.data.main = true;
+                console.log(model);
```

Comment:
If you're going to use `console.log`, you should wrap it in a `if (!Stratus.Environment.get('production'))` so it only shows in `dev` or `prev` environments.